UBSanIgnoreList.txt

```diff
@@ -0,0 +1 @@
+src:*/shared_ptr_base.h
```

Comment:
@junliume : Given the discussion on disabling MLIR in #3200 and #3198, we should remove this file or this line and place this PR on top of #3200 changes.